src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -63,14 +105,14 @@ class CallActor(call: Call, locallyQualifiedInputs: Map[String, WdlValue], backe
 
     def failedInstantiation(e: Throwable): Unit = {
       val message = s"Call '${call.fullyQualifiedName}' failed to launch command: " + e.getMessage
-      log.error(e, s"$tag: $message")
+      log.error(e, s"$tag: $call failed: $message")
       context.parent ! CallFailed(call, message)
     }
 
     def launchCall(commandLine: String): Unit = {
-      log.info(s"$tag: launching `$commandLine`")
+      log.info(s"$tag: launching ${call.name} for workflow ${workflowDescriptor.name} as `$commandLine`")
```

Comment:
Though I still think the workflow name is redundant... Also `call.fullyQualifiedName` is the way to uniquely identify a specific call within a workflow.